README.md

```diff
@@ -94,6 +94,15 @@ For Windows, you can use the following commands:
 * `gradlew spotlessApply` an check and apply formatting to any plain-text file.</br>
 * `gradlew build`  provides a command line to execute build script.</br>
 
+To get the latest apk fom the Github actions artifacts, follow these steps
+
+1. Navigate to the **Actions** tab of this repository.
+2. Click on your particular workflow from the workflows list.
+3. Scroll down to the **Artifacts** section and click the **mobile-wallet** hyperlink.
+4. The zip file download will start.
```

Comment:
I don't think this is a step that _needs to be done manually_, maybe we can get rid of it.

Reply:
@s-ayush2903 Can you please specify which step no. specifically among these.

Reply:
> I don't think this is a step that _needs to be done manually_, maybe we can get rid of it.

@s-ayush2903 Can you please specify which step no. specifically among these.